lib/coarse_op_preconditioned.cu

```diff
@@ -235,6 +237,10 @@ namespace quda {
     case 32: calculateYhat<storeFloat,Float,32>(Yhat, Xinv, Y, X); break;
     case 48: calculateYhat<storeFloat,Float,48>(Yhat, Xinv, Y, X); break;
     case 64: calculateYhat<storeFloat,Float,64>(Yhat, Xinv, Y, X); break;
+    case 96: calculateYhat<storeFloat,Float,96>(Yhat, Xinv, Y, X); break;
```

Comment:
How bad is compilation time for Yhat?

Reply:
Not sure, tbh. Never bad enough for me to complain ;) But I'll add some macro guards since they won't hurt.

Reply:
And I guess get rid of the unnecessary ones---we only need 12 (free field Wilson), 48 (Wilson and staggered), 64 (Wilson), 128 (staggered), and 192 (staggered).